libs/extensions/std/lang/src/gtfs-rt-interpreter-meta-inf.ts

```diff
@@ -0,0 +1,113 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+import { strict as assert } from 'assert';
+
+import {
+  BlockMetaInformation,
+  IOType,
+  PropertyAssignment,
+  PropertyValuetype,
+  isTextLiteral,
+} from '@jvalue/jayvee-language-server';
+import { ValidationAcceptor } from 'langium';
+
+export class GtfsRTInterpreterMetaInformation extends BlockMetaInformation {
+  constructor() {
+    super(
+      // How the block type should be called:
+      'GtfsRTInterpreter',
+
+      // Attribute definitions:
+      {
+        entity: {
+          type: PropertyValuetype.TEXT,
+          validation: isGtfsRTEntity,
+          docs: {
+            description: `Entity to process from GTFS-RT-feed (\`trip_update\`, \`alert\` or \`vehicle\`).
+            We currently support following Output-Sheets, each are an equivalent to the flattened Element Index defined in https://developers.google.com/transit/gtfs-realtime/reference#element-index (just required fields are included)):
```

Comment:
To make sure that markdown recognizes the link:
```suggestion
            We currently support following Output-Sheets, each are an equivalent to the flattened Element Index defined in <https://developers.google.com/transit/gtfs-realtime/reference#element-index> (just required fields are included)):
```